### PR TITLE
Feature/tc2 1293 romania documentation

### PR DESCRIPTION
**Pull request by Silvesb**

### CODE REVIEW

File: affiliates/romania.swagger.json [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+              "maxLength": 255
+            },
+            "last_name": {
+              "type": "string",
+              "example": "Iancu",
+              "maxLength": 255
+            },
+            "personal_id": {
+              "type": "string",
+              "description": "Format: /^(\\d{9}|\\d{12})$/",
```
Comment (kalmerkaurson) [inline affiliates/romania.swagger.json]: It should be only 13 digits long:
https://en.wikipedia.org/wiki/National_identification_number#Romania

File: affiliates/romania.swagger.json [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+            "email",
+            "first_name",
+            "last_name",
+            "personal_id",
+            "phone",
+            "agreements"
+          ],
+          "type": "object",
+          "properties": {
+            "first_name": {
```
Comment (kalmerkaurson) [inline affiliates/romania.swagger.json:254]: Should add regex format for "first_name" and "last_name"

Comment (Silvesb) [reply affiliates/romania.swagger.json:254]: added!

File: affiliates/romania.swagger.json (hunk 1 above)
Comment (Silvesb) [reply affiliates/romania.swagger.json]: now using a proper regex

File: affiliates/romania.swagger.json [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+              "type": "string",
+              "description": "Format: +$/uiA-Za-zĂÂÎŞŢăâîşţ]+[\\s\\-\\']?)+$/ui",
```
Comment (kalmerkaurson) [inline affiliates/romania.swagger.json:256]: a little broken i think

Comment (Silvesb) [inline affiliates/romania.swagger.json]: fixed!

**Review by Nuffic — APPROVED** [review]

**Comment by mikk150** [comment]
LGTM @kalmerkaurson plz review your suggestions

**Comment by kalmerkaurson** [comment]
@mikk150 I have no way to resolve or remove the comments

**Pull request merged**